spec/lang/syntax.md

```diff
@@ -239,12 +239,13 @@ pub enum Statement {
 pub enum Terminator {
     /// Just jump to the next block.
     Goto(BbName),
-    /// `condition` must evaluate to a `Value::Bool`.
-    /// If it is `true`, jump to `then_block`; else jump to `else_block`.
-    If {
-        condition: ValueExpr,
-        then_block: BbName,
-        else_block: BbName,
+    /// `value` needs to either evaluate to a `Value::Bool` or `Value::Int`.
```

Comment:
I think we want this to be only for `Int`. We can add a bool-to-int cast for the `bool` case.

Reply:
Implemented in the last commit.